packages/core/test/render3/matchers.ts

```diff
@@ -36,10 +36,10 @@ export function matchObjectShape<T>(
     }
     return true;
   };
-  matcher.jasmineToString = function() {
+  matcher.jasmineToString = function(pp: typeof jasmine.pp) {
     let errors: string[] = [];
     if (!_actual || typeof _actual !== 'object') {
-      return `Expecting ${jasmine.pp(expect)} got ${jasmine.pp(_actual)}`;
+      return `Expecting ${jasmine.pp(expect)} got ${pp(_actual)}`;
```

Comment:
@petebacondarwin did you intend for this to be

```suggestion
      return `Expecting ${pp(expect)} got ${pp(_actual)}`;
```

Reply:
Doh! Yes.